Extend FactRetriever model so we can represent them in the UI (#12474)
backstage/backstage: .changeset/rich-steaks-juggle.md

---
'@backstage/plugin-tech-insights-node': patch
'@backstage/plugin-tech-insights-backend': patch
---

> The FactRetriever model is extended by adding optional title and description fields. This allows us to display Fact
> Retrievers in the UI in future.

Reviewer: ```suggestion
The `FactRetriever` model is extended by adding optional title and description fields, allowing you to display them in the UI.
```
